NewEvaluation.css

```diff
@@ -0,0 +1,194 @@
+body{
```

Comment:
This file looks remarkably similar to `Evaluation.css`. What are the differences between the two? Which is the final version? I'll skip review of this _New_ file for now.